src/BreakpointViewer.cpp

```diff
@@ -521,20 +526,19 @@ void BreakpointViewer::stretchTable(BreakpointRef::Type type)
 	}
 }
 
-std::map<QString, BreakpointRef>::iterator
-BreakpointViewer::scanBreakpointRef(const Breakpoint& bp)
+BreakpointRef* BreakpointViewer::scanBreakpointRef(const Breakpoint& bp)
 {
 	BreakpointRef::Type type = bp.type == Breakpoint::BREAKPOINT ? BreakpointRef::BREAKPOINT
 	                        : (bp.type == Breakpoint::CONDITION  ? BreakpointRef::CONDITION : BreakpointRef::WATCHPOINT);
-	if (maps[type].empty()) return maps[type].end();
+	if (maps[type].empty()) return nullptr;
```

Comment:
No need to check `maps[type].empty()`, if it is empty then the code below acts as if the breakpoint is not found.
(And it's not faster to do this check as an optimization).

Reply:
Oops... I forgot about this one.